configure.ac

```diff
@@ -1403,10 +1403,6 @@ if test "x$use_coverage" = "xyes"; then
     AC_MSG_ERROR([GCC is required for --enable-coverage])
   fi
 
-  if test "$debug" != "yes"; then
-    AC_MSG_ERROR([--enable-debug is required for --enable-coverage])
-  fi
```

Comment:
Does coverage work without debug?

Reply:
I don't see any reason why it shouldn't.

Reply:
I see many reasons, but we don't need/use it so :+1: